### ISSUE TITLE
Checkboxes on Events tab are not labelled

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#536]
Form elements must have labels. The checkboxes on the Events tab (one per row and the check-all checkbox at the top) do not have them. This could be an implicit or explicit `<label>`, an `aria-label` or `title` attribute.

This has critical impact on blind/low vision people.
